src/main/java/org/squiddev/plethora/integration/hatchery/IntegrationHatchery.java

```diff
@@ -0,0 +1,114 @@
+	 * Feeder
+	 * Fertilized dirt
+	 * Fertilizer
+	 * Fertilizer Mixer
+	 * Generator
+	 * Manure
+	 * Nest
+	 * Nest pen
+	 * Nursery
+	 * Shredder
+	 * Chicken feed
+	 * Chicken manure
+	 * Fluid pump
+	 * Hatchery egg
+	 * ItemChickenMachine - ??
+	 * Prize Egg
+	 * Sprayer
+	 * Upgrades
+	 * ChickensHelper - may be a useful reference?
+	 */
+
+	public static final IMetaProvider<ItemStack> META_ANIMAL_NET = new ItemStackContextMetaProvider<AnimalNet>(
+			AnimalNet.class,
+			"Provides the entity captured inside this Animal Net."
+	) {
+
+		//TODO All three (so far) classes that expose captured entity data have many of the same checks;
+		// Is there any way that we can abstract the complexity/deduplicate the code?
+		@Nonnull
+		@Override
+		public Map<Object, Object> getMeta(@Nonnull IPartialContext<ItemStack> context, @Nonnull AnimalNet item) {
+			//Check for a captured entity
+			ItemStack netStack = context.getTarget();
+			if (!AnimalNet.hasCapturedAnimal(netStack)) return Collections.emptyMap();
+
+			//Check for an entity ID
+			NBTTagCompound entityData = AnimalNet.getCapturedAnimalNBT(netStack);
+			if (!entityData.hasKey("id", Constants.NBT.TAG_STRING)) return Collections.emptyMap();
+
+			//Check if we have a location
+			IWorldLocation location = context.getContext(ContextKeys.ORIGIN, IWorldLocation.class);
+			if (location == null) return getBasicDetails(entityData);
+
+			//Only the empty AnimalNet is registered, so we don't need to check for a generic
+			Entity entity = EntityList.createEntityFromNBT(entityData, location.getWorld());
+			if (entity == null) return getBasicDetails(entityData);
+
+			Vec3d loc = location.getLoc();
+			entity.setPositionAndRotation(loc.x, loc.y, loc.z, 0, 0);
+			return Collections.singletonMap("capturedEntity", context.makePartialChild(entity).getMeta());
+		}
+
+		//Copied wholesale from IntegrationThermalExpansion.META_MORB
+		private Map<Object, Object> getBasicDetails(NBTTagCompound entityData) {
+			String translationKey = EntityList.getTranslationName(new ResourceLocation(entityData.getString("id")));
+			if (translationKey == null) return Collections.emptyMap();
+
+			String translated = Helpers.translateToLocal("entity." + translationKey + ".name");
+
+			Map<Object, Object> details = new HashMap<>(2);
+			details.put("name", translated);
+			details.put("displayName",
+					entityData.hasKey("CustomName", Constants.NBT.TAG_STRING)
+							? entityData.getString("CustomName")
+							: translated
+			);
+			return Collections.singletonMap("capturedEntity", details);
+		}
+
+		@Nullable
+		@Override
+		public ItemStack getExample() {
+			//TODO Cannot cleanly generate an example, as the public methods on AnimalNet
+			// require a parameter of type EntityPlayer, and manually constructing the
+			// stack would be very brittle, as we would be relying entirely upon
+			// implementation details not changing...
+			//ModItems.animalNet.getDefaultInstance();
+			return null;
```

Comment:
You can technically use `EntityPlayerDummy`, which is what we do for the player meta providers:

https://github.com/SquidDev-CC/plethora/blob/ad8bbffeb134dda08c7bc6e5d3e7148848ff9b6d/src/main/java/org/squiddev/plethora/integration/vanilla/meta/MetaEntityPlayer.java#L42

However, it's your call - most meta providers don't have examples right now, so it's not the end of the world if this doesn't...